# Antecedent Enhancer Activity Predicts Future Susceptibility to Seizures in Mice

Pre-printed on bioRxiv as: 

Boros BD, Gachechiladze MA, Guo J, Galloway DA, Mueller SM, Shabsovich M, Yen A, Cammack AJ, Shen T, Mitra RD, Dougherty JD, Miller TM. Prior epigenetic status predicts future susceptibility to seizures in mice. bioRxiv [Preprint]. 2025 Mar 23:2025.03.20.644199. doi: https://doi.org/10.1101/2025.03.20.644199

## Authors

Boros BD, Gachechiladze MA, Guo J, Galloway DA, Mueller SM, Shabsovich M, Yen A, Chen X, Cammack AJ, Shen T, Mitra RD, Dougherty JD, Miller TM.

## Abstract

Wide variation of responses to identical stimuli presented to genetically inbred mice suggests the hypothesis that stochastic non-genetic variation, such as in chromatin state or enhancer activity during neurodevelopment, can mediate such phenotypic differences. However, this hypothesis is largely untested since capturing pre-existing molecular states requires non-destructive, longitudinal recording. Therefore, we tested the potential of Calling Cards (CC) to record transient neuronal enhancer activity during postnatal development in mice, and thereby associate such non-genetic variation with a subsequent phenotypic presentation â€“ degree of seizure response to the pro-convulsant pentylenetetrazol. We show that recorded differences in enhancer activity at 243 loci predict a severe vs. mild response, and that these are enriched near genes associated with human epilepsy. We also validated pharmacologically a seizure-modifying role for two previously unassociated genes, Htr1f and Let7c. This proof-of-principle supports using CC broadly to discover predisposition loci for other neuropsychiatric traits and behaviors.  Finally, as human disease is also influenced by non-genetic factors, similar epigenetic predispositions are possible in humans. 

## Data Availability

The raw and processed data used to generate these results are available on GEO at accession number [GSE305773](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE305773).

## Acknowledgements

The authors thank Harrison Gabel for helpful comments on the manuscript, the DNA Sequencing Innovation Lab, Genome Technology Access Center at the McDonnell Genome Institute (GTAC@MGI), and the High Throughput Computing Facility at the Center for Genome Sciences and Systems Biology. This work was supported by the Hope Center Viral Vectors Core at Washington University School of Medicine. This work was supported by grants from the National Institute of Mental Health (RF1MH117070, RF1MH126723 to R.D.M., J.D.D., and F30MH136688-01A1 to M.A.G.); National Institute of Aging (F30AG082394-01A1 to B.D.B.).
